src/commands/new.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
This file has a method that starts with `use*` and react-hooks/rules-of-hooks is complaining about that.

I'm not sure if this is the best approach here.